### PR TITLE
install_ffmpeg: Add FLV muxer

### PR DESCRIPTION
**Pull request by j0sh**
This should make RTMP output work on production (non-debug) builds of LPMS

### CODE REVIEW

**Comment by j0sh** [comment]
Tests are still broken in CI but pass locally

```
josh@josh-gpu:~/lpms$ time go test --tags=nvidia -timeout 1h ./ffmpeg
ok      github.com/livepeer/lpms/ffmpeg 1575.155s

real    26m16.094s
user    36m18.451s
sys     1m49.643s
```

File: install_ffmpeg.sh [hunk 1]
```diff
@@ -205,7 +205,7 @@ fi
 
 if [[ $BUILD_TAGS == *"debug-video"* ]]; then
   echo "video debug mode, building ffmpeg with tools, debug info and additional capabilities for running tests"
-  DEV_FFMPEG_FLAGS="--enable-muxer=md5,flv --enable-demuxer=hls --enable-filter=ssim,tinterlace --enable-encoder=wrapped_avframe,pcm_s16le "
```
Comment (victorges) [inline install_ffmpeg.sh:208]: Why remove this one? Are those only extra from the non-dev ones?

**Review by victorges — APPROVED** [review]
LGTM!

File: install_ffmpeg.sh (hunk 1 above)
Comment (j0sh) [reply install_ffmpeg.sh:208]: Yes, the base configuration includes flv now, so specifying it (again) with the dev flags isn't necessary (trying to keep things tidy)

**Pull request merged**